src/covid_shared/paths.py

```diff
@@ -12,6 +12,7 @@
 CITYMAPPER_MOBILITY_TEMPLATE = "https://cdn.citymapper.com/data/cmi/Citymapper_Mobility_Index_{DATE}.csv"
 OPEN_COVID19_GROUP_REPO = "https://github.com/beoutbreakprepared/nCoV2019/archive/master.zip"
 CDC_DEATHS_BY_RACE_ETHNICITY_AGE_STATE = "https://data.cdc.gov/api/views/ks3g-spdg/rows.csv?accessType=DOWNLOAD"
+SEROSURVEY_SUPPLEMENTAL_DATA = '/snfs1/Project/covid/data_intake/serology/supplemental_serosurvey_metadata'
```

Comment:
we don't want to use `nsfs1` here because that name is an underlying implementation detail. Infra could change it at any time. We want to use the symlink `/home/j` instead.